metashare/api/jobs.py

```diff
@@ -109,6 +155,13 @@ def _create_org_and_run_flow(scratch_org, *, user, repo_id, repo_branch, project
     scratch_org.last_modified_at = now()
     scratch_org.latest_revision_numbers = get_latest_revision_numbers(scratch_org)
 
+    scheduler = get_scheduler("default")
+    days = settings.DAYS_BEFORE_ORG_EXPIRY_TO_ALERT
+    before_expiry = scratch_org.expires_at - timedelta(days=days)
+    scheduler.enqueue_at(
+        before_expiry, alert_user_about_expiring_org, org=scratch_org, days=days,
```

Comment:
@wlonk This now also passes in `days`, so that the email is correct if the env var changes, and also since that env var may change between org creation (when this is scheduled) and when the email is sent out.

Reply:
This implies we should also allow repluralization of the email, and that makes me realize we should consider how to translate the email, which I think is a different story perhaps, as it doesn't fit inside any of our existing translation patterns.

Reply:
Couldn't we use the `blocktrans` template tag? https://docs.djangoproject.com/en/3.0/topics/i18n/translation/#std:templatetag-blocktrans

Or construct the body of the email in python instead of in a template.

Reply:
I guess we can use `blocktrans`; I wasn't thinking about it adequately. Adding.